packages/terra-base/src/Base.scss

```diff
@@ -13,7 +13,7 @@ body {
   background-color: var(--terra-base-background-color, #fff);
   color: var(--terra-base-color, #1c1f21);
   font-size: 1rem;
-  height: var(--terra-base-body-height, auto);
+  height: var(--terra-base-body-height, 100%);
```

Comment:
Please update consumer theme to have a body height of auto. This change is very non-passive to embedded content.

Reply:
With this change, can you add `terra-mixins` to dependencies / peerDependencies.

e.g.
```json
"peerDependencies": {
    "terra-mixins": "^1.10.0"
  },
  "dependencies": {
    "terra-mixins": "^1.10.0"
  },
```

And then we also need to import the mixins file in Base.scss and _theme.scss. Add this to the top of those files
```scss
@import '~terra-mixins';
```